nifi-registry/nifi-registry-core/nifi-registry-jetty/src/main/java/org/apache/nifi/registry/jetty/JettyServer.java

```diff
@@ -147,9 +151,39 @@ private File getDocsDir(final String docsDirectory) {
     }
 
     private void configureConnectors() {
-        final ServerConnectorFactory serverConnectorFactory = new ApplicationServerConnectorFactory(server, properties);
-        final ServerConnector serverConnector = serverConnectorFactory.getServerConnector();
-        server.addConnector(serverConnector);
+        try {
+            final ServerConnectorFactory serverConnectorFactory = new ApplicationServerConnectorFactory(server, properties);
+            final Map<String, String> interfaces = properties.isHTTPSConfigured() ? properties.getHttpsNetworkInterfaces() : Collections.emptyMap();
+            final Set<String> interfaceNames = interfaces.values().stream().filter(StringUtils::isNotBlank).collect(Collectors.toSet());
+            if (properties.isHTTPSConfigured() && !interfaceNames.isEmpty()) {
+                interfaceNames.stream()
+                        // Map interface name properties to Network Interfaces
+                        .map(interfaceName -> {
+                            try {
+                                logger.error("Checking {}", interfaceName);
```

Comment:
This error log should be removed.

Reply:
My bad! Forgot to remove after testing! Removing now